swap/src/monero/wallet.rs

```diff
@@ -195,8 +195,8 @@ impl Wallet {
             if proof.confirmations > confirmations.load(Ordering::SeqCst) {
                 confirmations.store(proof.confirmations, Ordering::SeqCst);
                 info!(
-                    "Monero lock tx received {} out of {} confirmations",
-                    proof.confirmations, expected_confirmations
+                    "Received {} out of {} confirmations for Monero lock tx {}",
+                    proof.confirmations, expected_confirmations, transfer_proof.tx_hash.0
```

Comment:
> Maybe even use tracing's value capture for it?

Just did a small test and yes we can!

```rust
use tracing::subscriber;
use tracing_subscriber::FmtSubscriber;

fn main() {
    let _guard = subscriber::set_default(tracing_subscriber::fmt()
        .with_writer(std::io::stderr)
        .with_ansi(true)
        .with_target(false)
        .finish());

    let txid = bitcoin::Txid::default();

    tracing::info!(%txid, "Bitcoin lock transaction has 1 out of 3 confirmations");
}
```

This prints the following:

```
Mar 04 14:40:02.514  INFO Bitcoin lock transaction has 1 out of 3 confirmations txid=0000000000000000000000000000000000000000000000000000000000000000
```

So I suggest we change the message to:

```
let txid = transfer_proof.tx_hash.0;
info!(%txid, "Monero lock tx has {} out of {} confirmations", proof.confirmations, expected_confirmations);
```

Reply:
Done, note that I had to make it a reference to avoid cloning the tx-hash.